frame/collective/src/benchmarking.rs

```diff
@@ -34,6 +34,27 @@ fn assert_last_event<T: Config<I>, I: 'static>(generic_event: <T as Config<I>>::
 	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
 }
 
+fn id_to_remark_data<T: Config<I>, I: 'static>(id: u32, length: usize) -> Vec<u8> {
+	let mut data: Vec<u8> = vec![];
+	let mut value = id.clone();
+	let mut value_size = 0;
+
+	while value > u8::MAX as u32 {
```

Comment:
@ggwpez I implemented something similar to the second solution from the previous comment. It is important to note that this doesn't cover every `u32` value(the provided `length` defines the maximum `MaxProposals` size that will work), but I tested it and this works with values up to around `2048` as `MaxProposals` and I believe that no greater value than this will ever be used for benchmarking since it will be really slow.

Reply:
Yea it looks a bit better than before. Will take a look tomorrow.